run_ejb_app_tests.sh

```diff
@@ -19,8 +19,9 @@ echo "LD_LIBRARY_PATH=${LD_LIBRARY_PATH}"
 
 cd exonum-java-binding-core/rust
 
-# Stable works well unless you want benchmarks.
-RUST_COMPILER_VERSION="stable"
+## Stable works well unless you want benchmarks.
+# TODO: stable does not work well until ECR-1839 is resolved
+RUST_COMPILER_VERSION="1.26.2"
```

Comment:
What if we make these scripts aware of the `RUST_VERSION` environment variable: if it's set use it, if it is not — use default?

Reply:
:thinking: but in this case it would be hard to run these scripts. You suggest to set `RUST_VERSION` in our dev envrironments (manually)?

Reply:
No, 
>  if it is not [set] — use default

We have already done this with JAVA_HOME, it may be done easily with this feature: https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

Reply:
Well, what do you mean by `default`? Stable? Rustup default? 1.26.2?

Reply:
Previously — stable, at this moment — 1.26.2.